src/components/table/cellRenderers/ClientCellRenderer.tsx

```diff
@@ -8,6 +8,8 @@ export const ClientCellRenderer = ({
 }) => {
   const { avatarImageUrl, email, name, fallbackColor } = value;
 
+  if (!name) return <></>;
+
```

Comment:
After the latest changes in previous PR, `client` can go undefined. Since you've applied similar technique [here](https://github.com/copilot-platforms/profile-manager/blame/239b60bca00023d3b61f2839869ea5573bd26906/src/components/table/cellRenderers/CompanyCellRenderer.tsx#L10) in the case of `company`, I followed the same approach. This should be good?

Reply:
Awesome, was just curious since I lost some context here.